content/en/docs/reference/kubectl/conventions.md

```diff
@@ -27,11 +26,15 @@ For a stable output in a script:
 For `kubectl run` to satisfy infrastructure as code:
 
 * Tag the image with a version-specific tag and don't move that tag to a new version. For example, use `:v1234`, `v1.2.3`, `r03062016-1-4`, rather than `:latest` (For more information, see [Best Practices for Configuration](/docs/concepts/configuration/overview/#container-images)).
-* Capture the parameters in a checked-in script, or at least use `--record` to annotate the created objects with the command line for an image that is lightly parameterized.
-* Pin to a specific [generator](#generators) version, such as `kubectl run --generator=run-pod/v1`.
 * Check in the script for an image that is heavily parameterized.
 * Switch to configuration files checked into source control for features that are needed, but not expressible via `kubectl run` flags.
 
+You can use the `--dry-run` flag to preview the object that would be sent to your cluster, without really submitting it.
+
+{{< note >}}
+Generators other than `pod` in `kubectl run` were deprecated until [kubernetes v1.17](https://v1-17.docs.kubernetes.io/docs/reference/kubectl/conventions/). Now there are not supported.
```

Comment:
```suggestion
All `kubectl` generators are deprecated.
```
?

Maybe move the deprecation note below the `#### Generators` heading as well?

Reply:
(sorry about the edits)

Reply:
done as requested.
```
commit 57d4d52817d1f6ee01ea406492a6111e71b3fd4c
Author: Marcin Niemira <marcin.niemira@gmail.com>
Date:   Tue Apr 21 20:40:49 2020 +1000

    change sentence to nicer form as requested during CR

diff --git a/content/en/docs/reference/kubectl/conventions.md b/content/en/docs/reference/kubectl/conventions.md
index ea72c3725..c4bdd59ec 100644
--- a/content/en/docs/reference/kubectl/conventions.md
+++ b/content/en/docs/reference/kubectl/conventions.md
@@ -32,7 +32,7 @@ For `kubectl run` to satisfy infrastructure as code:
 You can use the `--dry-run` flag to preview the object that would be sent to your cluster, without really submitting it.
 
 {{< note >}}
-Generators other than `pod` in `kubectl run` were deprecated until [kubernetes v1.17](https://v1-17.docs.kubernetes.io/docs/reference/kubectl/conventions/). Now there are not supported.
+All `kubectl` generators are deprecated. See the Kubernetes v1.17 documentation for a [list](https://v1-17.docs.kubernetes.io/docs/reference/kubectl/conventions/#generators) of generators and how they were used.
 {{< /note >}}
 
 #### Generators

```